lib/msal-browser/apiReview/msal-browser.api.md

```diff
@@ -993,6 +993,7 @@ export { INetworkModule }
 // @public
 export type InitializeApplicationRequest = {
     correlationId?: string;
+    isBroker?: boolean;
```

Comment:
Please don't make this part of public surface

Reply:
Isn't this autogenerated? You mean make it a private variable on the class?